src/view/sort-view.js

```diff
@@ -1,24 +1,45 @@
 import AbstractView from '../framework/view/abstract-view.js';
 
-const SORT_TYPES = ['day', 'event', 'time', 'price', 'offer'];
+const SortColumns = ['day', 'event', 'time', 'price', 'offer'];
+const ActiveSort = ['day', 'time', 'price'];
```

Comment:
[```suggestion
const ActiveSortTypes = ['day', 'time', 'price'];
```
Gthtvt](https://up.htmlacademy.ru/ecmascript/23/criteries#b8)